src/tests/class_tests/openms_gui/CMakeLists.txt

```diff
@@ -91,7 +97,7 @@ if(HAS_XSERVER)
     # add headers
     target_include_directories(${_gui_test} PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/include/GUI)
     # link against openms_gui and qt
-    target_link_libraries(${_gui_test} ${OpenMS_GUI_LIBRARIES} ${QT_LIBRARIES} Qt5::Test)
+    target_link_libraries(${_gui_test} ${OpenMS_GUI_LIBRARIES} ${QT_LIBRARIES} Qt5::Test Qt5::Widgets)
```

Comment:
What is in Openms GUI libraries and Qt libraries if not Qt Widgets. There must be something wrong.

Reply:
whoops.. yeah this is an artifact from an early version...
needs to be removed

Reply:
Cool. No worries